lib/transitive.js

```diff
@@ -178,6 +178,7 @@ export default class Transitive {
     if (!this.display) return
     const smWestSouth = sm.forward(llBounds[0])
     const smEastNorth = sm.forward(llBounds[1])
+    this.display.reset()
```

Comment:
Probably worth adding a comment here related to this issue. Otherwise, incredible work finding the needle in the haystack!